.circleci/config.yml

```diff
@@ -139,11 +139,11 @@ jobs:
             chmod +x /usr/local/bin/kubectl
             cp $(pwd)/artifacts/bin/okteto-Linux-x86_64 /usr/local/bin/okteto
              /usr/local/bin/okteto login --token ${API_STAGING_TOKEN}
+      - integration-deprecated
```

Comment:
Why do we have to run them first?

Reply:
It was just to get earlier feedback on the changes, not really necessary